cmake/test.cmake

```diff
@@ -356,6 +356,15 @@ finally:
         "MAYA_DISABLE_CIP=1"
         "MAYA_DISABLE_CER=1")
 
+    # Set a temporary folder path for the test
+    # Note: replace bad chars in test_name with _.
+    string(REGEX REPLACE "[:<>\|]" "_" SANITIZED_TEST_NAME ${test_name})
+    set(TEST_TEMP_DIR "${CMAKE_BINARY_DIR}/test/Temporary/${SANITIZED_TEST_NAME}")
+    set_property(TEST "${test_name}" APPEND PROPERTY ENVIRONMENT
+        "TMP=${TEST_TEMP_DIR}"
+        "TEMP=${TEST_TEMP_DIR}")
+    file(MAKE_DIRECTORY ${TEST_TEMP_DIR})
```

Comment:
I would suggest just adding these two lines (with your comment) to the env var above on line 340.
```
        "TMP=${MAYA_APP_TEMP_DIR}"
        "TEMP=${MAYA_APP_TEMP_DIR}")
```

Reply:
I will, nice catch.